Products/CMFPlomino/PlominoForm.py

```diff
@@ -1242,11 +1242,14 @@ def readInputs(self, doc, REQUEST, process_attachments=False, applyhidewhen=True
                                 doc,
```

Comment:
You are missing the code in validate which checks the tmp doc for a None if the field is mandatory.